macros/math/rand.sql

```diff
@@ -25,3 +25,9 @@
     random()
 
 {%- endmacro -%}
+
+{% macro redshift__rand() %}
+
+    random()
```

Comment:
Did you mean to include this in this PR? Or was this necessary for you to pass the tests?

Reply:
The default function `rand` doesn't exist in redshift so it was throwing an error. I use dbt with redshift that's why I ran into this issue!

Reply:
Gotcha, yeah this package doesn't support Redshift very well because I don't have a way to test there. 
I think what I'll do is take this out and move this to a separate PR, since this isn't directly related to the issue you opened. That way, I'll also better remember to add this to the dbt 0.20 branch.